frontend/src/component/pages/shared/PreferenceToggle.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component, ReactChild } from 'react';
+import { Toggle } from '../../ui/Toggle';
+
+interface IProps {
+  toggleLabel: ReactChild;
+  defaultIsEnabled: boolean;
+  onToggleClick?: (enabled: boolean) => void;
+}
+
+const DEFAULT_PROPS = {
+  defaultIsEnabled: false
+};
+
+export class PreferenceToggle extends Component<IProps> {
+  static defaultProps: Partial<IProps> = DEFAULT_PROPS;
+
+  private renderLabel() {
+    return <>{this.props.toggleLabel}</>;
+  }
+
+  render() {
+    return (
+      <div className={'creation-toggle'}>
+        <Toggle
+          defaultIsEnabled={this.props.defaultIsEnabled}
+          onClick={this.props.onToggleClick}
+        ></Toggle>
+        <span className={'toggle-label'}>{this.renderLabel()}</span>
```

Comment:
<div>{this.props.toggleLabel}</div>

Please use flex layout for `preference-toggle`.

Reply:
I think it already is... 
![image](https://user-images.githubusercontent.com/3276350/80824948-6b0f7680-8bad-11ea-8398-005cf6240652.png)